synth.py

```diff
@@ -98,6 +138,13 @@
   1
 )
 
+os.rename("dev/.gitignore", ".gitignore")
```

Comment:
Not sure what all goes here, but this may be missing `.prettierignore`

Reply:
Added

Reply:
I take that back: That file doesn't exist:

FileNotFoundError: [Errno 2] No such file or directory: 'dev/.prettierignore' -> '.prettierignore'

I guess we should add this to our templates?